python/example/single_cell_recipe.py

```diff
@@ -1,13 +1,35 @@
+#!/usr/bin/env python3
+
 import arbor
-import numpy, pandas, seaborn # You may have to pip install these.
+import pandas, seaborn # You may have to pip install these.
 
 # The corresponding generic recipe version of `single_cell_model.py`.
 
-# (1) Define a recipe for a single cell and set of probes upon it.
+# (1) Create a morphology with a single (cylindrical) segment of length=diameter=6 μm
+tree = arbor.segment_tree()
+tree.append(arbor.mnpos, arbor.mpoint(-3, 0, 0, 3), arbor.mpoint(3, 0, 0, 3), tag=1)
+
+# (2) Define the soma and its center
+labels = arbor.label_dict({'soma':   '(tag 1)',
+                           'center': '(location 0 0.5)'})
+
+# (3) Create cell and set properties
+decor = arbor.decor()
+decor.set_property(Vm=-40)
+decor.paint('"soma"', 'hh')
+decor.place('"center"', arbor.iclamp( 10, 2, 0.8))
+decor.place('"center"', arbor.spike_detector(-10))
+
+# (4) Create cell and the single cell model based on it
+cell = arbor.cable_cell(tree, labels, decor)
```

Comment:
Comment on line 23 mentions 'single cell model', which probably isn't correct for this example.

Reply:
Correct(ed)!